pyodide-build/pyodide_build/buildall.py

```diff
@@ -559,10 +556,10 @@ def copy_packages_to_dist_dir(
     packages: Iterable[BasePackage], output_dir: Path
 ) -> None:
     for pkg in packages:
-        try:
-            shutil.copy(pkg.wheel_path(), output_dir)
-        except RuntimeError:
-            pass
+        if pkg.library:
+            continue
+
```

Comment:
Much better. This `except RuntimeError: pass` is pretty horrendous (so I probably wrote it...)